Podcast/PlayerControlsView.swift

```diff
@@ -76,54 +92,90 @@ class PlayerControlsView: UIView {
+        }
         
         recommendButton = RecommendButton(frame: CGRect(x: marginSpacing, y: self.frame.maxY - buttonsYInset, width: recommendButtonSize.width, height: recommendButtonSize.height))
         recommendButton.addTarget(self, action: #selector(recommendButtonTapped), for: .touchUpInside)
-        addSubview(recommendButton)
+        
+        nextButton = UIButton(frame: .zero)
+        nextButton.setBackgroundImage(#imageLiteral(resourceName: "next"), for: .normal)
+        nextButton.adjustsImageWhenHighlighted = false
+        nextButton.addTarget(self, action: #selector(nextButtonPress), for: .touchUpInside)
+        addSubview(nextButton)
+        nextButton.snp.makeConstraints { make in
+            make.size.equalTo(nextButtonSize)
+            make.centerY.equalTo(forwardsButton.snp.centerY)
+            make.trailing.equalTo(slider.snp.trailing)
+        }
+        
+        speedButton = UIButton(frame: .zero)
+        speedButton.titleLabel?.font = ._12RegularFont()
+        speedButton.setTitleColor(.slateGrey, for: .normal)
+        speedButton.addTarget(self, action: #selector(speedButtonPress), for: .touchUpInside)
+        addSubview(speedButton)
+        speedButton.snp.makeConstraints { make in
+            make.size.equalTo(speedButtonSize)
+            make.leading.equalToSuperview().offset(marginSpacing)
+            make.centerY.equalTo(forwardsButton.snp.centerY)
+        }
         
         moreButton = MoreButton(frame: .zero)
-        moreButton.frame.size = moreButtonSize
         moreButton.frame.origin = CGPoint(x: frame.maxX - marginSpacing - moreButtonSize.width, y: self.frame.maxY - buttonsYInset)
         moreButton.addTarget(self, action: #selector(moreButtonTapped), for: .touchUpInside)
         addSubview(moreButton)
+        moreButton.snp.makeConstraints { make in
+            make.size.equalTo(moreButtonSize)
+            make.bottom.equalToSuperview().inset(moreButtonBottomOffset)
+            make.trailing.equalToSuperview().inset(marginSpacing)
+        }
         
-        updateUI(isPlaying: false, elapsedTime: "0:00", timeLeft: "0:00", progress: 0.0, isScrubbing: false)
+        updateUI(isPlaying: false, elapsedTime: "0:00", timeLeft: "0:00", progress: 0.0, isScrubbing: false, rate: .normal)
     }
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
-    func updateUI(isPlaying: Bool, elapsedTime: String, timeLeft: String, progress: Float, isScrubbing: Bool) {
+    func updateUI(isPlaying: Bool, elapsedTime: String, timeLeft: String, progress: Float, isScrubbing: Bool, rate: PlayerRate) {
         if isPlaying {
-            playPauseButton.setBackgroundImage(#imageLiteral(resourceName: "player_pause_icon"), for: .normal)
+            playPauseButton.setBackgroundImage(#imageLiteral(resourceName: "pause"), for: .normal)
```

Comment:
again, with this, lets make the pause button the selected state and the player button the normal state (feel free to abstract this away to a PlayerPlayPauseButton class if you think the logic here gets wonky)